src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) p.6

Reply:
Even I wanna reset some properties?

Reply:
@Mykola-Maik I was not right about this one, sorry for that. Did not notice you're resetting. Indeed you can do that for resetting, my bad